src/ncp/ncp_base.hpp

```diff
@@ -82,6 +88,15 @@ class NcpBase
      */
     explicit NcpBase(Instance *aInstance);
 
+    /**
```

Comment:
Sme here? add this under `OPENTHREAD_CONFIG_MULTIPLE_INSTANCE_ENABLE?`
(we basically want to support multiple OT instance config in `NcpBase`)?